src/demos/calcite-dropdown.html

```diff
@@ -308,6 +308,17 @@ <h3>Not calcite-button triggers</h3>
         <calcite-dropdown-item>Table</calcite-dropdown-item>
       </calcite-dropdown-group>
     </calcite-dropdown>
+    <br />
+
+    <calcite-dropdown>
+      <input slot="dropdown-trigger" placeholder="I'm an input dropdown-trigger, and the carat is too!" />
```

Comment:
Suuuper picky, but have a little misspelling here.
```suggestion
      <input slot="dropdown-trigger" placeholder="I'm an input dropdown-trigger, and the caret is too!" />
```

Reply:
Funny, I initially misspelled it as the icon's attribute value and fixed it there, forgetting I'd written it here too. Thanks!